.pre-commit-config.yaml

```diff
@@ -1,12 +1,12 @@
 repos:
-- repo: https://github.com/pycqa/flake8
-  rev: 4.0.1
-  hooks:
-  - id: flake8
-- repo: https://github.com/omnilib/ufmt
-  rev: v1.3.0
-  hooks:
-    - id: ufmt
-      additional_dependencies:
-        - black == 21.4b2
-        - usort == 0.6.4
+  - repo: https://github.com/pycqa/flake8
+    rev: 4.0.1
+    hooks:
+      - id: flake8
+  - repo: https://github.com/omnilib/ufmt
+    rev: v1.3.0
+    hooks:
+      - id: ufmt
+        additional_dependencies:
+          - black == 21.4b2
+          - usort == 0.6.4
```

Comment:
Looks like we are adding whitespace here. Maybe we can exclude the changes to this file for this PR to make things easier?